bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -124,7 +124,7 @@ class App extends Component {
 
     return (
       <Resizable
-        minWidth="10%"
+        minWidth="13%"
```

Comment:
To be fair those values shouldn`t be in %, they should be in pixel.

Reply:
Ok I will test it changing % to px, thanks

Reply:
![bandicam-2018-05-23-14-05-34-522](https://user-images.githubusercontent.com/26368353/40442991-f0cd4ff8-5e92-11e8-9347-e10dcbf52bf6.gif)